src/deploy/index.ts

```diff
@@ -67,11 +67,6 @@ export const deploy = async function (
       await prepareFrameworks(targetNames, context, options);
       const nowTargetsFunctions = targetNames.includes("functions");
       if (nowTargetsFunctions && !usedToTargetFunctions) {
-        if (context.hostingChannel && !experiments.isEnabled("pintags")) {
```

Comment:
Let's revert this change for now and capture enabling this experiment by default in a follow on, SG?